lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -56,6 +60,10 @@ def violated_rules(class_name)
           /\A#{deprecated_rule[:class_expr]}\z/.match(class_name)
         end
       end
+
+      def pad_with_left_space(message)
+        message ? " #{message}" : ''
```

Comment:
I'm letting the user define the suggestion and addendum messages without a leading left space. However when I'm concatenating them together in the final message I need to both of them with a leading space if they are defined, otherwise I can leave them as empty strings.